src/modules/InviteUser/components/invitationActions.vue

```diff
@@ -0,0 +1,153 @@
+<script>
+import { reInviteUser, revokeUser } from "../services";
+export default {
+    props: {
+        name: {
+            type: String,
+            required: true,
+        },
+        email: {
+            type: String,
+            required: true,
+        },
+        contact:{
+          type:String,
+          required:true
+        }
+        
+    },
+    methods: {
+      bodyData(){
+        return   {
+                  token:"1bxXVTgMmdgiClqXZ8Rdmg",
+                  data:{
+                    name: this.name,
+                    email: this.email,
+                }}
+              
+      },
+        async reInviteMember() {
+            try {
+                const response = await reInviteUser(this.bodyData());
+                //display message coming from response
+                console.log(response.data)
+            }
+            catch (error) {
+                console.error("error:", error.message);
+            }
+        },
+        async revokeMember() {
+            try {
+                const response = await revokeUser(this.bodyData());
+                //display message coming from response
+                console.log(response.data);
+
+            }
+            catch (error) {
+                console.error("error:", error);
```

Comment:
remove "console.log"

Reply:
A toast or a dialog box may be appropriate but since it has not been implemented console.log is used to check for error right now. will implement toast or dialog box in upcoming sprint